serverless-dev.yml

```diff
@@ -0,0 +1,22 @@
+service: ${file(./serverless-base.yml):service}
+frameworkVersion: ${file(./serverless-base.yml):frameworkVersion}
+package: ${file(./serverless-base.yml):package}
+functions: ${file(./serverless-base.yml):functions}
+
+provider:
+  name: ${file(./serverless-base.yml):provider.name}
+  runtime: ${file(./serverless-base.yml):provider.runtime}
+  stage: dev
+
+plugins:
+  - serverless-esbuild
+  - serverless-offline
+
+custom:
+  serverless-offline:
+    httpPort: ${env:PORT, 3100}
+    noPrependStageInUrl: true
```

Comment:
I wonder if either of these options are relevant, considering there's nothing to hit.